pkg/testing/kubernetes/supported.go

```diff
@@ -72,6 +72,14 @@ var variants = []struct {
 		Name:  "service",
 		Image: "docker.elastic.co/beats-ci/elastic-agent-service",
 	},
+	{
+		Name:  "edot-collector",
```

Comment:
If this has the same test coverage as the ELASTIC_AGENT_OTEL=true tests that exist already, instead of using edot in their place as https://github.com/elastic/elastic-agent/pull/7173/files#r1999460268 suggest perhaps we just need to delete those extra tests.